### ISSUE TITLE
File not found on Xcode build

### ISSUE THREAD

**Issue by swvcc** [Telerik-Verified-Plugins/WKWebView#40]
I'm receiving this error when I attempt to build. I followed the directions precisely with the cordova add from the git and the cordova prepare command.

CompileC /Users/swvcc/Library/Developer/Xcode/DerivedData/cmsmobile-fpldwbmpvcljdpawxtrnfsvoxoud/Build/Intermediates/cmsmobile.build/Debug-iphoneos/cmsmobile.build/Objects-normal/arm64/MyMainViewController.o cmsmobile/Plugins/com.telerik.plugins.wkwebview/MyMainViewController.m normal arm64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
    cd /Users/swvcc/Desktop/approval-app-2.1/2.1-new/cordova/platforms/ios
    export LANG=en_US.US-ASCII
    export PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin"
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -x objective-c -arch arm64 -fmessage-length=0 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit=0 -std=c99 -fobjc-arc -Wno-trigraphs -fpascal-strings -O0 -Wno-missing-field-initializers -Wno-missing-prototypes -Wno-implicit-atomic-properties -Wno-receiver-is-weak -Wno-arc-repeated-use-of-weak -Wduplicate-method-match -Wno-missing-braces -Wparentheses -Wswitch -Wunused-function -Wno-unused-label -Wno-unused-parameter -Wunused-variable -Wunused-value -Wempty-body -Wuninitialized -Wno-unknown-pragmas -Wno-shadow -Wno-four-char-constants -Wno-conversion -Wconstant-conversion -Wint-conversion -Wbool-conversion -Wenum-conversion -Wshorten-64-to-32 -Wpointer-sign -Wno-newline-eof -Wno-selector -Wno-strict-selector-match -Wundeclared-selector -Wno-deprecated-implementations -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS8.1.sdk -fstrict-aliasing -Wprotocol -Wdeprecated-declarations -g -fvisibility=hidden -Wno-sign-conversion -miphoneos-version-min=6.0 -iquote /Users/swvcc/Library/Developer/Xcode/DerivedData/cmsmobile-fpldwbmpvcljdpawxtrnfsvoxoud/Build/Intermediates/cmsmobile.build/Debug-iphoneos/cmsmobile.build/Endeavour-generated-files.hmap -I/Users/swvcc/Library/Developer/Xcode/DerivedData/cmsmobile-fpldwbmpvcljdpawxtrnfsvoxoud/Build/Intermediates/cmsmobile.build/Debug-iphoneos/cmsmobile.build/Endeavour-own-target-headers.hmap -I/Users/swvcc/Library/Developer/Xcode/DerivedData/cmsmobile-fpldwbmpvcljdpawxtrnfsvoxoud/Build/Intermediates/cmsmobile.build/Debug-iphoneos/cmsmobile.build/Endeavour-all-target-headers.hmap -iquote /Users/swvcc/Library/Developer/Xcode/DerivedData/cmsmobile-fpldwbmpvcljdpawxtrnfsvoxoud/Build/Intermediates/cmsmobile.build/Debug-iphoneos/cmsmobile.build/Endeavour-project-headers.hmap -I/Users/swvcc/Library/Developer/Xcode/DerivedData/cmsmobile-fpldwbmpvcljdpawxtrnfsvoxoud/Build/Products/Debug-iphoneos/include -I/Users/swvcc/Library/Developer/Xcode/DerivedData/cmsmobile-fpldwbmpvcljdpawxtrnfsvoxoud/Build/Products/Debug-iphoneos/usr/local/lib/include -I/Users/swvcc/Library/Developer/Xcode/DerivedData/cmsmobile-fpldwbmpvcljdpawxtrnfsvoxoud/Build/Intermediates/UninstalledProducts/include -I/Users/swvcc/Library/Developer/Xcode/DerivedData/cmsmobile-fpldwbmpvcljdpawxtrnfsvoxoud/Build/Products/Debug-iphoneos -I/Users/swvcc/Library/Developer/Xcode/DerivedData/cmsmobile-fpldwbmpvcljdpawxtrnfsvoxoud/Build/Intermediates/cmsmobile.build/Debug-iphoneos/cmsmobile.build/DerivedSources/arm64 -I/Users/swvcc/Library/Developer/Xcode/DerivedData/cmsmobile-fpldwbmpvcljdpawxtrnfsvoxoud/Build/Intermediates/cmsmobile.build/Debug-iphoneos/cmsmobile.build/DerivedSources -F/Users/swvcc/Library/Developer/Xcode/DerivedData/cmsmobile-fpldwbmpvcljdpawxtrnfsvoxoud/Build/Products/Debug-iphoneos -include /Users/swvcc/Library/Developer/Xcode/DerivedData/cmsmobile-fpldwbmpvcljdpawxtrnfsvoxoud/Build/Intermediates/PrecompiledHeaders/cmsmobile-Prefix-fntlirgzvwdalscjgytovceuhpdi/cmsmobile-Prefix.pch -MMD -MT dependencies -MF /Users/swvcc/Library/Developer/Xcode/DerivedData/cmsmobile-fpldwbmpvcljdpawxtrnfsvoxoud/Build/Intermediates/cmsmobile.build/Debug-iphoneos/cmsmobile.build/Objects-normal/arm64/MyMainViewController.d --serialize-diagnostics /Users/swvcc/Library/Developer/Xcode/DerivedData/cmsmobile-fpldwbmpvcljdpawxtrnfsvoxoud/Build/Intermediates/cmsmobile.build/Debug-iphoneos/cmsmobile.build/Objects-normal/arm64/MyMainViewController.dia -c /Users/swvcc/Desktop/approval-app-2.1/2.1-new/cordova/platforms/ios/cmsmobile/Plugins/com.telerik.plugins.wkwebview/MyMainViewController.m -o /Users/swvcc/Library/Developer/Xcode/DerivedData/cmsmobile-fpldwbmpvcljdpawxtrnfsvoxoud/Build/Intermediates/cmsmobile.build/Debug-iphoneos/cmsmobile.build/Objects-normal/arm64/MyMainViewController.o

clang: error: no such file or directory: '/Users/swvcc/Desktop/approval-app-2.1/2.1-new/cordova/platforms/ios/cmsmobile/Plugins/com.telerik.plugins.wkwebview/MyMainViewController.m'
clang: error: no input files
Command /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang failed with exit code 1

**Comment by etiennea** [comment]
yes I have this too and this sux because if I remove the target and rebuild it it works but I have to redo all the apple watch stuff :-/
